MapboxNavigation/NavigationMapView.swift

```diff
@@ -1040,13 +1039,14 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             return
         }
         
-        // Sadly, `MGLMapView.setVisibleCoordinateBounds(:edgePadding:animated:)` uses the current pitch and direction of the mapview. Ideally, we'd be able to pass in zero.
+        // Sadly, `MGLMapView.cameraThatFitsShape(_:direction:edgePadding:)` uses the current pitch of the mapview.
+        // Because of this, we need to set it before or it will create a camera for the current pitch instead of 0.
         let camera = self.camera
         camera.pitch = 0
-        camera.heading = 0
```

Comment:
Minor improvement. We are now force setting only one camera property before calculating the correct camera for the line.